extensions/job_management/standard/requirements/provenance/run/REQ_get-op.adoc

```diff
@@ -0,0 +1,8 @@
+[[req_job-provenance_run_get-op]]
+[requirement]
+====
+[%metadata]
+label:: /req/provenance/run/get-op
+part:: For every created jobs (path: `/jobs/{jobId}/run`), the server SHALL support the HTTP GET operation.
```

Comment:
What is this endpoint meant to do? We don't have that in openEO... Must it be required or can it be optional?

Reply:
This endpoint is to fulfill requirements regarding provenance (the name of the endpoint is disputable). The idea would be to return https://en.wikipedia.org/wiki/PROV_(Provenance) metadata. 

I think could accept alternate content negotiation on `GET /jobs/{jobID}` directly (similar to the `GET /jobs/{jobID}/definition` comment) instead of this extra path. 

One thing that would be helpful is to have this PROV definition required. This way, we can finally have a common referencing mechanism of data/processing provenance. If optional, we will never end up having a common provenance resolution, since backends will always have different job representations and APIs. And if optional, nobody will bother implementing it...